src/main/kotlin/rules/ConversionRule.kt

```diff
@@ -0,0 +1,17 @@
+package com.eigenholser.flac2mp3.rules
+
+import org.jeasy.rules.api.Fact
+import org.jeasy.rules.api.Facts
+import org.jeasy.rules.api.Rule
+
+interface ConversionRule: Rule {
+    override fun execute(facts: Facts) = facts.add(Fact(name, true))
+
+    override fun compareTo(other: Rule): Int {
+        return when {
+            this.priority > other.priority -> 1
+            this.priority < other.priority -> -1
+            else -> this.name.compareTo(other.name)
+        }
+    }
```

Comment:
Who is calling this? Upon close examination, this.priority == other.priority in all tests. Consequently it is always returning 0. Do we have need for this anywhere?

Reply:
There is only one rule. Ergo, nothing to compare. Try removing the function and see what happens. :grin: